.github/workflows/add_comment.yml

```diff
@@ -0,0 +1,89 @@
+name: Add "bors r+" comment
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  add_label:
+    name: Add "bors r+" comment
+    if: github.event.label.name == 'ready-to-merge'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: octokit/request-action@v2.x
+        name: Get comments
+        id: get_comments
+        with:
+          route: GET /repos/:repository/issues/:pull_number/comments
+          repository: ${{ github.repository }}
+          pull_number: ${{ github.event.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # Parse steps.get_comments.outputs.data, since it is a string
+      - id: parse_comments
+        name: Parse comments
+        run: |
+          RESULT=$(jq -c --raw-output 'map(select(.body | test("\\n?bors:? (r\\+|merge)"; "m"))) | length' <<'EOF'
+          ${{ steps.get_comments.outputs.data }}
+          EOF
+          )
```

Comment:
I would appreciate a review of these lines to make sure they're sufficiently secure. I'm hoping it's not possible for a user to input a PR comment that will escape this...

Reply:
I believe it's fine since a JSON document may not contain the line `EOF`.